srcpkgs/python-discogs_client/files/LICENSE

```diff
@@ -0,0 +1,26 @@
+Copyright © 2015 Discogs. All rights reserved.
+
```

Comment:
Why including the license? You could just download it from github...
https://raw.githubusercontent.com/discogs/discogs_client/master/LICENSE

Reply:
I'm downloading the package from PyPI, and the tar doesn't have a LICENSE; I made a search in the repository and found out others solved this problem by sending the template **and** the LICENSE file together, so I did the same.

Could it be in `distfiles` instead?

Reply:
Done.

> You could just download [the license] from github

Maybe this should be made to other packages as well...

Reply:
https://travis-ci.org/void-linux/void-packages/jobs/423733185#L870

you forgot `skip_extraction="LICENSE"`